docs/advanced-features/measuring-performance.md

```diff
@@ -28,7 +30,22 @@ function MyApp({ Component, pageProps }) {
 export default MyApp
 ```
 
-This function is fired when the final values for any of the metrics have finished calculating on
+Alternatively you can use the `useWebVitalsReport` hook API, with a callback function to receive metric reports:
+
+```js
+// pages/home.js
+import { useWebVitalsReport } from 'next/vitals'
+
+export default function Home() {
+  useWebVitalsReport((metric) => {
+    console.log(metric)
+  })
+
+  return <div>Home Page</div>
+}
+```
+
+The exported `reportWebVitals` function or the callback parameter of `useWebVitalsReport` is fired when the final values for any of the metrics have finished calculating on
```

Comment:
> final

I'm not sure this is strictly true -- e.g. `web-vitals` will trigger CLS again if you close the page.

Reply:
replaced it with "essential"